xml/System.Data.SqlClient/SqlColumnEncryptionKeyStoreProvider.xml

```diff
@@ -14,7 +14,7 @@
   </Base>
   <Interfaces />
   <Docs>
-    <summary>Base class for all key store providers. A custom provider must derive from this class and override its member functions and then register it using SqlConnection.RegisterColumnEncryptionKeyStoreProviders(). For details see, [Always Encrypted](/sql/relational-databases/security/encryption/always-encrypted-database-engine).</summary>
+    <summary>Base class for all key store providers. A custom provider must derive from this class and override its member functions and then register it using SqlConnection.RegisterColumnEncryptionKeyStoreProviders(). For details see, <see href="https://docs.microsoft.com/sql/relational-databases/security/encryption/always-encrypted-database-engine">Always Encrypted</see>.</summary>
     <remarks>To be added.</remarks>
     <related type="ExternalDocumentation" href="http://blogs.msdn.com/b/sqlsecurity/archive/tags/always+encrypted/">Always Encrypted Blog</related>
     <related type="ExternalDocumentation" href="https://msdn.microsoft.com/library/mt147923.aspx">Always Encrypted (client development)</related>
```

Comment:
nit unrelated to your PR: this could also be changed to link to docs: \<related type="Article" href="/sql/relational-databases/security/encryption/always-encrypted-database-engine">

Reply:
Not sure what you mean by this comment @rpetrusha - are you saying that line 20 could be a docs link? Or that the 'for details' sentence in the summary would be better being moved to the related articles?

Reply:
Sorry, @ChrisMaddock. You're right; <https://docs.microsoft.com/en-us/sql/relational-databases/security/encryption/always-encrypted-client-development> is the correct URL; I have no idea where I came up with the other URL. Also, relative links for \<related> tags are OK.

Reply:
Great - thanks! Remaining changes made!